packages/dds/merge-tree/src/test/client.applyMsg.spec.ts

```diff
@@ -1015,6 +1015,46 @@ describe("client.applyMsg", () => {
 
 			logger.validate({ baseText: "BBBBBB B" });
 		});
+		// TODO: see AB#29398
+		it.skip("obliterate with mismatched final states", () => {
+			const clients = createClientsAtInitialState(
+				{
+					initialState: "0{zzzzzzz}123{yyyyyy}45",
+					options: {
+						mergeTreeEnableObliterate: true,
+						mergeTreeEnableSidedObliterate: true,
+						mergeTreeEnableAnnotateAdjust: true,
+					},
+				},
+				"A",
+				"B",
+			);
+			let seq = 0;
+			const logger = new TestClientLogger(clients.all);
+			const ops: ISequencedDocumentMessage[] = [];
+			const b = clients.B;
+
+			ops.push(
+				b.makeOpMessage(
+					b.obliterateRangeLocal({ pos: 1, side: Side.After }, { pos: 9, side: Side.Before }),
+					++seq,
+				),
+				b.makeOpMessage(b.insertTextLocal(2, "66"), ++seq),
```

Comment:
You could also use different text here to distinguish from the later 6s.
```suggestion
				b.makeOpMessage(b.insertTextLocal(2, "xx"), ++seq),
```